test/runtests.jl

```diff
@@ -12,7 +12,7 @@ using Test, MixedLayerThermoclineDynamics
 
     @test test_dx(grid1D, dx)
     @test test_xF(grid1D, range(0, stop = Lx - dx, length = nx))
-    @test xdomain_length(grid1D, Lx - 0)
+    @test xdomain_length(grid1D, Lx)
```

Comment:
This is good but it's irrelevant with this PR. It's good to try to have self-contained PRs. Imagine the scenario you want to undo the PR for some reason, then this test for grids will go back to this `Lx-0` weirdness. :)

Reply:
I'll keep this in mind from now on.